history/titles/c_dawning_lane.txt

```diff
@@ -4,4 +4,15 @@
 	law = succ_primogeniture
 }
 10.1.1={holder = 42002}
-603.1.30={holder=42003}
\ No newline at end of file
+603.1.30={holder=42003}
+603.9.1={
+    effect = {
+        location = {
+			add_province_modifier = { name = depopulated_3 duration = -1 }
+		}
+	}
+}
+604.1.1={
+	holder = 42200		# Patron character
+	liege = k_quelthalas
```

Comment:
but already has `liege=e_quelthalas` above?

Reply:
Fixed in 8d5699cfab69b950f50bdbd82ed471e9959d6ff9.